modules/ROOT/pages/changelogs.adoc

```diff
@@ -8,6 +8,8 @@
 [source, status codes, role="noheader"]
 -----
 Neo.ClientNotification.Cluster.RequestedTopologyMatchedCurrentTopology
+Neo.ClientNotification.Schema.IndexOrConstraintAlreadyExists
```

Comment:
Since 5.17 is next I've added them to that for now, but will update with whatever version the neo4j PR ends up in

Reply:
made the cut-off